Makefile

```diff
@@ -207,8 +207,6 @@ install: precheck
 	ln -sf $(SM_DEST)lcache.py $(SM_STAGING)$(BIN_DEST)tapdisk-cache-stats
 	ln -sf /dev/null $(SM_STAGING)$(UDEV_RULES_DIR)/69-dm-lvm-metad.rules
 	install -m 755 scripts/xs-mpath-scsidev.sh $(SM_STAGING)$(UDEV_SCRIPTS_DIR)
-	install -m 755 scripts/xe-getarrayidentifier $(SM_STAGING)$(BIN_DEST)
```

Comment:
Sorry, these need removing from mk/sm.spec.in as well. XenServer doesn't use this file but it's provided as a template for other users and needs to be kept up to date.